scaleway/resource_function_cron_test.go

```diff
@@ -77,6 +78,73 @@ func TestAccScalewayFunctionCron_Basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckScalewayFunctionCronExists(tt, "scaleway_function_cron.main"),
 					resource.TestCheckResourceAttr("scaleway_function_cron.main", "schedule", "0 0 * * *"),
+					resource.TestCheckResourceAttr("scaleway_function_cron.main", "name", "tf-tests-cron-basic"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccScalewayFunctionCron_NameUpdate(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy:      testAccCheckScalewayFunctionCronDestroy(tt),
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_function_namespace main {
+						name = "tf-tests-function-cron-basic"
```

Comment:
The name should reflect the current test, this name reflect the basic test
```suggestion
						name = "tf-tests-function-cron-name-update"
```